### ISSUE TITLE
Total population grows 

### ISSUE THREAD

**Issue by bwolfson97** [bwolfson97/UCLA-Networks-Final-Project#10]
Normally in prisons, the number of inmates coming in equals the number of inmates going out (otherwise the prison would grow infinitely large). 

We could make a background release rate equal to the birth rate, ensuring the total population remains constant. Then, the population would only change when the actual release intervention event occurs.

Additionally, all added inmates are currently susceptible. If we remove inmates regularly, they would come from any state, but be replaced with all susceptible inmates. This would unnaturally lower the # of infected. Instead, added inmates' states should represent the breakdown of states in the general non-prison population. (i.e. if 10% of people outside prison are infected, 10% of new inmates should be infected).

So, to recap, the proposed changes are:
1) create background release rate equal to birth rate 
2) make new inmates' state reflect state breakdown of non-prison population
